src/packages/frontend/account/licenses/managed-licenses.tsx

```diff
@@ -75,11 +73,15 @@ export const ManagedLicenses: React.FC = () => {
     if (licenses.size == 0) {
       return <div>You are not the manager of any licenses yet.</div>;
     }
+
+    const site_licenses: SiteLicenses = licenses.toJS().reduce((acc, v) => {
```

Comment:
This code is hard for me to read.  Yes, I know what it does, but... ugh.  Why not just be more straightforward in this case?